tests/unittests/OperatorTest.cpp

```diff
@@ -2213,6 +2213,78 @@ TEST_P(Operator, NonSquarePaddingConvolution) {
   EXPECT_TRUE(result.isEqual(result1));
 }
 
+/// Check non-square padding for AveragePool. The first pool op has non-square
+/// padding, while the second one has zero padding. The second pool op's input
+/// is the same as the first one's after-padding input. All other parameters of
+/// the two convs are the same.
+TEST_P(Operator, NonSquarePaddingAveragePool) {
+  auto *input = mod_.createVariable(ElemKind::FloatTy, {1, 4, 4, 1}, "input");
+  auto IH = input->getHandle();
+  for (size_t i = 0; i < 4 * 4; i++) {
+    IH.raw(i) = i + 1;
+  }
+  auto *Pool = F_->createPoolAvg("Pool", input, 2, 1, {0, 2, 1, 3});
+  SaveNode *S = F_->createSave("save", Pool);
```

Comment:
nit: I'd prefer consistently use either auto or [X]Node, otherwise it seems like there's a deliberate reason for the inconsistency. Same for the other test case as well.